extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/HttpUpgradeSecurityCheck.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.websockets.next.runtime;
+
+import static io.netty.handler.codec.http.HttpHeaderNames.CACHE_CONTROL;
+import static io.netty.handler.codec.http.HttpHeaderNames.LOCATION;
+
+import java.util.List;
+import java.util.Map;
+
+import io.quarkus.security.ForbiddenException;
+import io.quarkus.security.spi.runtime.MethodDescription;
+import io.quarkus.security.spi.runtime.SecurityCheck;
+import io.quarkus.websockets.next.HttpUpgradeCheck;
+import io.smallrye.mutiny.Uni;
+
+public class HttpUpgradeSecurityCheck implements HttpUpgradeCheck {
```

Comment:
I think that `SecurityHttpUpgradeCheck` would be a little bit more conveninent...

Reply:
I'll rename it.

Reply:
Done.